cli/cli.go

```diff
@@ -90,14 +90,13 @@ func InitCtx(cfg *Config, authn bool) *HttpContext {
 		cfg.Log.Err("Error: no target set\n")
 		return nil
 	}
-	tgt := cfg.Targets[cfg.CurrentTarget]
-	ctx := NewHttpContext(cfg.Log, tgt.Host, vidmBasePath, vidmBaseMediaType)
+	ctx := NewHttpContext(cfg.Log, cfg.Option(HostOption), vidmBasePath, vidmBaseMediaType)
 	if authn {
-		if token := cfg.Targets[cfg.CurrentTarget].AccessToken; token == "" {
+		if token := cfg.Option("accesstoken"); token == "" {
 			cfg.Log.Err("No access token saved for current target. Please log in.\n")
 			return nil
 		} else {
-			ctx.Authorization(cfg.Targets[cfg.CurrentTarget].AccessTokenType + " " + token)
+			ctx.Authorization(cfg.Option("accesstokentype") + " " + token)
```

Comment:
I am fine with this change but I am worried that now the strings for the options are repeated everywhere ("accesstoken", etc...). Should we define constants for all current options today?